lib/readmeBuilder.js

```diff
@@ -58,8 +58,16 @@ function build({ readme = true }) {
 
       const bytype = (type) => flist(pipe(
         schemas,
-        filter((schema) => schema[keyword`type`] === type), // remove schemas without matching type
-        filter((schema) => !!schema[s.parent]), // keep only schemas with a parent
+        filter((schema) => {
+          if (
+            schema[keyword`type`] === type // remove schemas without matching type
+            && !!schema[s.parent] // keep only schemas with a parent
+            && !schema.$ref // it is not a reference
+          ) {
+            return true;
+          }
+          return false;
+        }),
```

Comment:
```suggestion
         filter((schema) => schema[keyword`type`] === type), // remove schemas without matching type
         filter((schema) => !!schema[s.parent]), // keep only schemas with a parent
         filter((schema) => !schema.$ref), // it is not a reference
```

Good idea to filter references, but better to keep the existing filter chain given that it already works like `&&`